core/modules/taxonomy/taxonomy.module

```diff
@@ -65,8 +65,8 @@ function taxonomy_permission() {
  */
 function taxonomy_vocabulary_list_permissions($vocabulary) {
   $permissions = array(
-    'edit terms in ' . $vocabulary->machine_name => array(
-      'title' => t('%vocabulary: Edit terms', array('%vocabulary' => $vocabulary->name)),
```

Comment:
The array key needs to change (obviously), but I don't think the 'title' should change. I like the current matching of the '*Tags*: Edit terms' wording with the 'Edit' tab on the term page. Renaming this to 'Update' would create a discrepancy I think...

Reply:
What I mean is that the 'Edit terms' permission directly controls a user's access to the 'Edit' tab. So calling the permission 'Update terms' and having it control access to the 'Edit' tab could be confusing...

Reply:
Good point, let's stick with "Edit terms" in titles.